README.md

```diff
@@ -25,12 +30,106 @@ Social, Advertising, or Analytics categories as described
+and experiments. The lists currently in active use are:
+* `social-tracking-protection-blacklist.json`: a subset of social trackers from
+    Disconnect's blocklist. This list is used to identify "social media"
+    trackers within Firefox's UI. All of the origins on this list should also
+    be included in Disconnect's `disconnect-blacklist.json` list.
+* `google-mapping.json`: a legacy entity remapping file used by Firefox iOS.
+    This list should not be used be any new products. Instead, the
+    [`disconnect-mapping.json`](https://github.com/mozilla-services/shavar-list-creation/blob/master/disconnect_mapping.json)
+    list should be used.
+
+## List Versioning and Release Process
+
+As of Firefox 72, all desktop releases use versioned blocklists, i.e., each
+version of Firefox uses a version of `disconnect-blacklist.json` and
+`disconnect-entitylist.json` specific to that version. These versions are
+tracked by [branches](https://github.com/mozilla-services/shavar-prod-lists/branches)
+of this repository. For the current cycle (Dec. 2019) this means there is a
+73 list (Nightly), a 72 list (Beta), a 71 list (Release), and a 68 list (ESR).
+
+Nightly uses a staging version of the blocklist; the staging blocklist pulls in
+changes from Disconnect as soon as they are available. When a new version of
+Firefox is released, we will also release a new version of the list that
+corresponds to the version of Firefox moving from Nightly (main branch) --> Beta
+(versioned branch). That version of the list will ride the trains along with its
+respective Firefox version. Releases older than Firefox 69 use the 69 version of
+the blocklist.
+
+This means that all changes will be tested for at least the full beta cycle and
+part of the Nightly cycle. We may choose to shorten the testing cycle in the
+future.
+
+There are three possible exceptions to this process:
+1. **Fast-tracked changes** which are deployed immediately to all channels
+2. **Temporary exceptions** which are deployed using Remote Settings
+3. **List freezes** for when we’d like to test changes for a longer duration.
+   These are tracked in Github issues on this repository.
+
+#### Fast-tracked changes
+
+We will fast track breakage-related updates or policy-related updates, both
+of which may only be done by Disconnect. Fast-tracked changes should have
+minimal, if any, risk of breakage.
+
+Changes that may be fast-tracked include:
+* Deleting a domain from the list.
+* Adding new domains to the entity list.
+* Replacing a domain currently on the list with a new domain at the request of
+  the entity.
+* Moving a domain between list categories of the same feature.
+
+As soon as Disconnect makes changes of this type we will merge
+them into each versioned list and deploy them across all channels.
+
+#### Temporary exceptions
+
+If we discover breakage that we'd like to fix through Firefox code changes, we
+may temporarily whitelist the relevant domains while we work on the fixes. To
```

Comment:
```suggestion
may temporarily add the relevant domains to the entity list in Remote Settings while we work on the fixes. To
```

Reply:
This is not true, but I'll update the whitelist terminology. This is not related to the entity list.